client/blocks/comment-detail/index.jsx

```diff
@@ -233,7 +236,9 @@ export class CommentDetail extends Component {
 				className={ classes }
 				tabIndex="0"
 			>
-				<QueryComment commentId={ commentId } siteId={ siteId } />
+				{ ! preventServerRequest &&
+					<QueryComment commentId={ commentId } siteId={ siteId } />
```

Comment:
Let's keep an eye on this as something we might be able to tidy later. If we see a pattern, we may be able to update the data-layer to address this.